### ISSUE TITLE
what is alosaur.json use for?

### ISSUE THREAD

**Issue by Jordan-Hall** [alosaur/cli#3]
Can i ask what does alosaur.json do? 

I've noticed in the generator you have alosaur.json but wondering what the purpose of it was? I guess if you generate multiple "apps", you going to want to stack it a bit like angular.cli or nx workspace.json or am I completely wrong?

Referencing https://github.com/alosaur/alosaur/issues/66 about custom DI. I think the CLI would go to be a full wrapper around Deno, including running, so it's self-executing.  After much research looks like Deno does support transpiler, so I was thinking to add it as a CLI support, so it has an AST compile and runs the compiled code instead. This way alosaur can add transformer support without it being an issue (for them that want too) Not sure if this takes away from what you were thinking though

Suggested change: 

```json
 {
   "apps": [
     {
       "name": "example-app",
       "permissions": [ "--allow-net", "--allow-read" ]
     }
   ]
 }
```

That way you can have:
```bash
deno run --allow-read --allow-write --allow-net cli.ts run example-app
```

**Comment by irustm** [comment]
Yeah, you thought right. The Alosaur CLI will be a wrapper over deno.run, and maybe there will be a way to run multiple applications.
But I don't have a clear roadmap for the Alosaur CLI yet. I am thinking of forming that roadmap this week.

If you have more ideas about CLI, please write about them
